core/pouch.js

```diff
@@ -228,6 +228,12 @@ class Pouch {
     })
   }
 
+  async allByRemoteIds (remoteIds /*: * */) /* Promise<Metadata[]> */ {
+    const params = {keys: Array.from(remoteIds), include_docs: true}
```

Comment:
mmmh, may be safer to split the array into a constant maximum batch size. But this can take place when we limit the change feed. Add a TODO ?